website/docs/cdktf/release/index.mdx

```diff
@@ -22,3 +22,20 @@ CDKTF includes upgrade guides for releases that have breaking changes.
 CDKTF uses feature flags to enable potentially breaking behaviors in a release. Feature flags are disabled by default, so existing projects that do not specify the flag will continue to work as expected with later CDKTF releases. New projects created using `cdktf init` include flags enabling all features available in the release that created the project.
 
 Edit the [`cdktf.json` configuration file](/cdktf/create-and-deploy/configuration-file#enable-features) to disable flags for which you prefer the old behavior, or to add flags to enable new behaviors after upgrading. The CHANGELOG contains a description of new feature flags in each release, and the CDKTF source file [features.ts](https://github.com/hashicorp/terraform-cdk/blob/main/packages/cdktf/lib/features.ts) provides a complete list of all feature flags.
+
+All feature flags are disabled if not set explicitly.
+
+### `excludeStackIdFromLogicalIds`
```

Comment:
Are these two things feature flags for the next release? If they are, I wonder if we can remove them, as we tell folks that we list all feature flags and descriptions in the changelog. Thoughts?